README.md

```diff
@@ -77,7 +120,32 @@ NAME                                         DESIRED   CURRENT   READY   AGE
 replicaset.apps/botkube-botkube-58c4579b44   1         1         1       7h55m
 ```
 
-🚀 Troubleshooting: [FluxCD Troubleshooting](https://fluxcd.io/flux/cheatsheets/troubleshooting/)
+For functional testing you need to create a **testJob**. See *eks-anywhere-addons/eks-anywhere-common/Testers* for examples. The descriptions about [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/) from upstream kubernetes provides helpful information and additional details.
+
+Presumably your application will provide a service that can be accessed to verify the application deployed as expected. In the test job access the service as basic functional verification. The generic access pattern to access a service in the cluster is `servicename.namespace.svc.cluster.local:port` where `servicename` is the name of your service, `namespace` is the namespace you assigned in your deployment yaml description and `port` is the port number your service is running on. If you are uncertain about the service name you can use
+
+```bash
+kubectl get services -n $NAMESPACE
+```
+
+where *$NAMESAPCE* is the namespace you assigned in your deployment yaml description. Jobs in kubernetes are immutable and as such whenever you modify the test job and before you run another
```

Comment:
This again is very generic to kubernetes and we dont want this here.

Reply:
Well, `git clone` is very generic as well. I had help figuring this out especially since I was contributing an application where I do not contribute to the development effort. I think it is good to be a little more explicit. Not everyone knows the kubectl command line inside and out.

Is there really harm in being a bit more explicit?